docker/tool-box/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM fedora:latest
 
 ENV PACKAGE_LIST git ansible tree vim unzip
-ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.6.173.0.21/linux/oc.tar.gz
+ENV OC_CLIENT_MIRROR https://mirror.openshift.com/pub/openshift-v3/clients/3.7.23/linux/oc.tar.gz
```

Comment:
Any chance we can adopt this one-liner instead of the many curl/tar/rm, etc below
https://github.com/redhat-cop/openshift-management/blob/master/images/prune-ocp-projects/Dockerfile#L10

Reply:
yea I can do that. will update shortly